src/backend/ceph.rs

```diff
@@ -511,14 +517,14 @@ impl CephBackend {
             debug!("Found tags for logical volume: {:?}", tags);
             let id_tag = tags.iter().find(|t| t.starts_with("ceph.osd_id"));
             if let Some(tag) = id_tag {
-                let parts: Vec<String> = tag.split('=').map(|s| s.to_string()).collect();
+                let parts: Vec<String> = tag.split('=').map(ToString::to_string).collect();
                 if let Some(s) = parts.get(1) {
                     osd_id = Some(u64::from_str(s)?);
                 }
             }
             let fsid_tag = tags.iter().find(|t| t.starts_with("ceph.osd_fsid"));
             if let Some(tag) = fsid_tag {
-                let parts: Vec<String> = tag.split('=').map(|s| s.to_string()).collect();
+                let parts: Vec<String> = tag.split('=').map(ToString::to_string).collect();
```

Comment:
Interesting.  I didn't know you could do this.

Reply:
Clippy!